docs/changes/latest.inc

```diff
@@ -21,6 +21,8 @@ Enhancements
 
 - Update docs to make it more uniform (by `Kaustubh Patil`_). 
 
+- Update model_selection and add capabilities to register searchers (by `Sami Hamdan`_).
```

Comment:
If with `model_selection` you refer to a module, make it a link.

Reply:
Is this correct: 
 :mod:\`.model_selection\`

Reply:
Would be something like that. Does it work?